site/slices/contact-us-slice/form/index.js

```diff
@@ -31,42 +31,47 @@ class Form extends Component {
     return (
       <section className={styles.formContainer} id="contactUs">
         <h2 className={styles.heading}>
-          We don’t have a sales team.
+          Project in mind?
           <br />
-          Speak to one of our founders.
+          Tell us more.
         </h2>
 
         <form className={styles.contactUsForm}>
-          <label className={styles.formLabel} htmlFor="message">
-            Message:&nbsp;
-            <span className={styles.errorMessage}>{errors.message}</span>
-          </label>
-          <textarea
-            rows="5"
-            className={cx({
-              inputBox: true,
-              hasErrors: errors.message,
-            })}
-            name="message"
-            id="contactUsMessage"
-            defaultValue={message}
-            onChange={this.handleInputChange}
-          />
+          <div className={styles.inputContainer}>
+            <label className={styles.formLabel} htmlFor="message">
+              Message:&nbsp;
+              <span className={styles.errorMessage}>{errors.message}</span>
+            </label>
+            <textarea
+              rows="5"
+              className={cx({
+                inputBox: true,
+                hasErrors: errors.message,
+              })}
+              name="message"
+              id="contactUsMessage"
+              defaultValue={message}
+              onChange={this.handleInputChange}
+            />
+          </div>
 
-          <label className={styles.formLabel} htmlFor="contact">
-            Your email:&nbsp;
-            <span className={styles.errorMessage}>{errors.contact}</span>
-          </label>
-          <input
-            className={cx({
-              inputBox: true,
-              hasErrors: errors.contact,
-            })}
-            name="contact"
-            type="text"
-            value={contact}
-            onChange={this.handleInputChange}
-          />
+          <div className={styles.inputContainer}>
+            <label className={styles.formLabel} htmlFor="contact">
+              Your email:&nbsp;
+              <span className={styles.errorMessage}>{errors.contact}</span>
+            </label>
+            <input
+              className={cx({
+                inputBox: true,
+                hasErrors: errors.contact,
+              })}
+              name="contact"
+              type="text"
+              onChange={this.handleInputChange}
+              value={contact}
+              placeholder="sam@company.com"
```

Comment:
@dstodolny , also around this email field, the text is rendered in black on Chrome, making it look less like a placeholder. Is it possible to make this consistent to how it looks in Firefox please? Examples below:

On Chrome:
<img width="519" alt="screen shot 2017-02-06 at 13 54 34" src="https://cloud.githubusercontent.com/assets/20296527/22649662/ddc79734-ec73-11e6-9c7d-9e7e5912790a.png">

On Firefox
<img width="523" alt="screen shot 2017-02-06 at 13 54 26" src="https://cloud.githubusercontent.com/assets/20296527/22649711/1a4ce0a6-ec74-11e6-9ecd-ebc2b3f163c5.png">